MarkupsToModel/MRML/vtkMRMLMarkupsToModelNode.cxx

```diff
@@ -159,6 +164,19 @@ void vtkMRMLMarkupsToModelNode::ReadXMLAttributes( const char** atts )
       }
       SetInterpolationType(interpolationType);
     }
+    else if ( ! strcmp( attName, "PointParameterType" ) )
+    {
+      int pointParameterType = 0;
+      std::stringstream nameString;
+      nameString << attValue;
+      nameString >> pointParameterType;
```

Comment:
I'm sorry, you are right. Github just showed a diff and I didn't see the string<->enum conversion.